Orbitersdk/samples/ProjectApollo/src_launch/mccvessel.h

```diff
@@ -24,6 +24,7 @@
 #pragma once
 
 #include "Orbitersdk.h"
+#include "connector.h"
```

Comment:
Is this needed or is it a remnant before you started using clbkGeneric?

Reply:
That's a remnant. I will remove it.